shared/nav.desktop.js

```diff
@@ -127,6 +128,16 @@ class Nav extends Component {
   }
 
   shouldComponentUpdate (nextProps, nextState) {
+    if (!this._checkedBootstrap) {
```

Comment:
we check if we've bootstrapped once. if we've not been provisioned we show the main window. This assumes the tab is the login tab etc, which is true, but fragile. i think it might make more sense (but a lot more code changing) to move this window showing logic outside of the nav